src/library/Server/Manager/Whm.php

```diff
@@ -83,6 +84,25 @@ public function testConnection()
         return true;
     }
 
+    // https://docs.cpanel.net/knowledge-base/accounts/reserved-invalid-and-misconfigured-username/
+    public function generateUsername($domainName)
+    {
+        $procsessedDomain = strtolower(preg_replace('/[^A-Za-z0-9]/', '', $domainName));
+        $username = substr($procsessedDomain, 0, 7) . random_int(0, 9);
```

Comment:
`$processedDomain`, no? 😛

Reply:
Huh, I'm a bit surprised cspell didn't flag that.
Might be a config issue 👀 

Well, that's to look at later